### DISCUSSION TITLE
Testcontainers Modules did not find SQL server

### DISCUSSION THREAD

**Discussion by wh-zxt-01** [testcontainers/testcontainers-java#9526] [General]
Is it still possible to use testcontainers for testing if there is no integrated database in them,

**Comment by eddumelendez** [comment]
Testcontainers provides a module for SQL Server. See https://testcontainers.com/modules/mssql/ and [docs](https://java.testcontainers.org/modules/databases/mssqlserver/).

If there is no module provided for the technology you are looking for, then use GenericContainer.
